jre_emul/Makefile

```diff
@@ -206,8 +206,8 @@ dist_includes: $(BUILD_DIR)/.jre_dist_includes
 $(BUILD_DIR)/.jre_dist_includes: $(PUBLIC_TRANSLATED_HEADERS:%=$(GEN_OBJC_DIR)/%) \
     $(PUBLIC_EMULATION_HEADERS:%=$(EMULATION_CLASS_DIR)/%) | $(DIST_INCLUDE_DIR)
 	@echo copying jre_emul header files
-	@(cd $(EMULATION_CLASS_DIR) && tar cf - \
-	    $(PUBLIC_EMULATION_HEADERS:$(EMULATION_CLASS_DIR)/%=%)) \
+	@cd $(EMULATION_CLASS_DIR) && tar cf - \
```

Comment:
So this was the only header copying command that gave you issues? How strange.

At first glance it seems that removing the sub shell would leave the shell in a different directory (EMULATION_CLASS_DIR) when entering the sub shell on line 211. I believe this would have been the original reason for the sub shell. Does this not affect the final destination of the headers?